setup.py

```diff
@@ -170,6 +215,24 @@ def get_extensions():
         for mod in ("mpi_xla_bridge", "mpi_xla_bridge_cpu")
     ]
 
+    if sycl_info["compile"] and sycl_info["libdirs"]:
+        extensions.append(
+            Extension(
+                name=f"{CYTHON_SUBMODULE_NAME}.mpi_xla_bridge_xpu",
+                sources=[f"{CYTHON_SUBMODULE_PATH}/mpi_xla_bridge_xpu.pyx"],
+                include_dirs=sycl_info["compile"],
+                library_dirs=sycl_info["libdirs"],
+                libraries=sycl_info["libs"],
+                language="c++",
+                define_macros=[("OMPI_SKIP_MPICXX", "1")],
+            )
+        )
+    else:
+        print_warning(
+            "SYCL (Intel Basekit) path not found. Did you call {you basekit dir}/setvars.sh? You can use env var ONEAPI_ROOT to point Basekit directory where sycl is",
```

Comment:
Let's be more neutral here, this typically doesn't imply a problem.

```suggestion
            "SYCL (Intel Basekit) path not found",
```